### ISSUE TITLE
Question :)

### ISSUE THREAD

**Issue by metabsd** [OneIdentity/ansible-authentication-services#4]
The unjoin process will delete the computer object in Active Directory ?

Really great work. I will use this collection in the next couple day 👍

**Comment by llnagy76** [comment]
Yes. The unjoin process removes your computer object from Active Directory.
